File: C99/백준/Bronze/10798. 세로읽기/세로읽기.c
#include <stdio.h>
#include <string.h>

int main() {
    char arr[5][16] = {0}; // 최대 15글자 + null
    for (int i = 0; i < 5; i++) {
        scanf("%s", arr[i]);
    }

    for (int col = 0; col < 15; col++) {
        for (int row = 0; row < 5; row++) {
            if (col < strlen(arr[row])) {
                printf("%c", arr[row][col]);
            }
        }
    }
    printf("\n");
    return 0;
}

File: Java/백준/Bronze/10798. 세로읽기/세로읽기.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String[] arr = new String[5];
        for (int i = 0; i < 5; i++) {
            arr[i] = sc.nextLine();
        }

        StringBuilder sb = new StringBuilder();
        for (int col = 0; col < 15; col++) {
            for (int row = 0; row < 5; row++) {
                if (col < arr[row].length()) {
                    sb.append(arr[row].charAt(col));
                }
            }
        }
        System.out.println(sb.toString());
    }
}
